edudoc/disciplinary-crash-course/disciplinary-crash-course.md

```diff
@@ -0,0 +1,98 @@
+# Guidelines for Disciplinary Incidents
+
+## Disciplinary
```

Comment:
This isn't really a useful section title, so I think we can remove it entirely. Then the following paragraph works just like an intro paragraph.